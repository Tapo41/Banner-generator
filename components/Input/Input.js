import styled from "styled-components"

const Input = styled.input`
  font-weight: 300;
  font-size: 1rem;
  padding: 0.3125rem;
  border-radius: 4px;
  border: 1px solid black;

  &:focus {
    outline: transparent;
    border: 1px solid darkgrey;
  }
`

export default Input
