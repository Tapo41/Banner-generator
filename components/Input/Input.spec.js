import React from 'react'
import { render } from 'react-testing-library'

import Input from "./Input"

describe("<Input />", () => {
  describe("Snapshots", () => {
    it("should match snapshot", () => {
      const { container } = render(<Input />);
      expect(container).toMatchSnapshot();
    });
  })
})
