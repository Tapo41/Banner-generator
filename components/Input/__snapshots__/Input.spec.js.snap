// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Input /> Snapshots should match snapshot 1`] = `
<div>
  <input
    class="sc-bdVaJa eQjeEW"
  />
</div>
`;
